org.taskcoach.TaskCoach.yaml

```diff
@@ -0,0 +1,191 @@
+app-id: org.taskcoach.TaskCoach
+runtime: org.gnome.Platform
+runtime-version: '3.30'
+sdk: org.gnome.Sdk
+command: taskcoach
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
```

Comment:
AFAIK it's still needed to pass `--socket=wayland`, and you should probably set `required-flatpak` too as it will be broken for pre-`fallback-x11` releases.